.ci.yaml

```diff
@@ -201,6 +201,7 @@ targets:
       - os=Linux
     dimensions:
       kvm: "1"
+    bringup: true
```

Comment:
Instead, we need to increase the `timeout` if https://github.com/flutter/engine/pull/50058 is intended. Otherwise this will just keeping flaking in staging due to timeout.